core/src/main/scala/com/databricks/labs/remorph/parsers/FunctionBuilder.scala

```diff
@@ -206,33 +221,45 @@ object FunctionBuilder {
     case "UPPER" => Some(FixedArity(1))
     case "USER_ID" => Some(VariableArity(0, 1))
     case "USER_NAME" => Some(VariableArity(0, 1))
+    case "VALUE" => Some(FixedArity(2, XmlFunction))
     case "XACT_STATE" => Some(FixedArity(0))
     case "YEAR" => Some(FixedArity(1))
     case _ => None
   }
   // GCOVR_EXCL_STOP
 
+  def functionType(name: String): FunctionType = {
+    val uName = name.toUpperCase(Locale.getDefault())
+    val defnOption = functionArity(uName)
+    defnOption match {
+      case Some(fixedArity: FixedArity) => fixedArity.functionType
+      case Some(variableArity: VariableArity) => variableArity.functionType
```

Comment:
```suggestion
      case Some(FixedArity(_ , functionType, _)) => functionType
      case Some(VariableArity(_, _, functionType, _)) => functionType
```

Reply:
I cannot do this as it is forbidden in the style guide. It was actually the way I first implemented it

Reply:
TIL